<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EMG</class>
 <widget class="QMainWindow" name="EMG">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1332</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EMG</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_graficos">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>19</y>
      <width>971</width>
      <height>691</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCustomPlot" name="grafico_at" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>871</width>
       <height>271</height>
      </rect>
     </property>
     <zorder>grafico_af</zorder>
     <zorder>grafico_af</zorder>
    </widget>
    <widget class="QCustomPlot" name="grafico_af" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>390</y>
       <width>871</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelSignal_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Amplitud / Tiempo</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSignal_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>360</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Amplitud / Frecuencia</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_opciones">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>691</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="botonArchivo">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Archivo</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSignal">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>26</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>(Vacio....)</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>271</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="BoxFiltros">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Pasa alto</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pasa bajo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pasa banda</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>14</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>FILTROS :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>FC</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>FMAX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>FA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botonAplicar">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Aplicar</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>271</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>MÃ©todos    :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboMetodo">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>dft</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fft</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sgram</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dgt</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>wdst</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dwt</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="botonEnviar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>251</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Graficar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botonCerrar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Cerrar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botonColorMap">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>colorMap</string>
      </property>
     </widget>
    </widget>
    <widget class="QPlainTextEdit" name="inputCom">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>490</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="botonRO">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>530</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Run Octave</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="outputCom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>271</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1332</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>info</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEdit"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
